test/test_entrypoint.py

```diff
@@ -27,8 +33,39 @@ def test_trivial_argparse():
 
 def test_mocked_cc_pipeline(tmpdir):
     args = parse_args([str(tmpdir), '-a', 'aarch64', '-o', 'ubuntu'])
-    with patch('ros_cross_compile.ros_cross_compile.DockerClient', Mock()) as docker_mock:
+    with patch('ros_cross_compile.ros_cross_compile.DockerClient', Mock()) as docker_mock,\
+            patch('ros_cross_compile.ros_cross_compile.check_install_script_exists') as sc_mock:
         cross_compile_pipeline(args)
+        assert sc_mock.called
         assert docker_mock.called
         assert docker_mock().build_image.call_count == 2
         assert docker_mock().run_container.call_count == 2
+
+
+def test_install_rosdep_script_exist(tmpdir):
+
+    contents = """
```

Comment:
Like we discussed offline, let's simplify by removing these contents, and do `data_file.touch()` below, instead of `write_text`